tests/components/asuswrt/test_sensor.py

```diff
@@ -323,32 +207,35 @@ async def test_temperature_sensors(
     "side_effect",
     [OSError, None],
 )
-async def test_connect_fail(hass: HomeAssistant, side_effect) -> None:
+async def test_connect_fail(hass: HomeAssistant, connect_legacy, side_effect) -> None:
     """Test AsusWRT connect fail."""
 
     # init config entry
     config_entry = MockConfigEntry(
         domain=DOMAIN,
-        data=CONFIG_DATA,
+        data=CONFIG_DATA_TELNET,
     )
     config_entry.add_to_hass(hass)
 
-    with patch(ASUSWRT_LIB) as asus_wrt:
-        asus_wrt.return_value.connection.async_connect = AsyncMock(
-            side_effect=side_effect
-        )
-        asus_wrt.return_value.async_get_nvram = AsyncMock()
-        asus_wrt.return_value.is_connected = False
+    connect_legacy.return_value.connection.async_connect = AsyncMock(
+        side_effect=side_effect
+    )
+    connect_legacy.return_value.async_get_nvram = AsyncMock()
```

Comment:
Is this really required?

Reply:
No it isn't, I don''t remember why it was here😉.